components/organisms/TeamSection/TeamSection.tsx

```diff
@@ -0,0 +1,112 @@
+import alexis from "@/public/Alexis.png";
+import shedrack from "@/public/shedrack.png";
+import Link from "next/link";
+
+const teamData = [
+  {
+    userName: "Shedrack Akintayo",
+    position: "Founder",
+    imgUrl: "shedrack.png",
+  },
+  {
+    userName: "Seunbayo Adebayo",
+    position: "Advocacy",
+    imgUrl: "",
+  },
+  {
+    userName: "Amarachi Iheanacho",
+    position: "Technical Writer",
+    imgUrl: "",
+  },
+  {
+    userName: "Damilola Ezekiel",
+    position: "Technical Writer",
+    imgUrl: "",
+  },
+  {
+    userName: "Alexis Abakasanga",
+    position: "Designer",
+    imgUrl: "alexis.png",
+  },
+];
+export default function TeamSection() {
+  function Card({ imgSource, title, position }) {
+    return (
+      <div className="flex flex-col gap-2">
+        <div className="bg-[#99999980] h-[21em] md:w-[22em]">
+          <img className="object-cover h-[21em]" src={imgSource} alt="" />
+        </div>
+        <div className="flex gap-4">
+          <p>{title}</p>
+          <div className="flex gap-4">
+            <Link href="#">
+              <svg
+                width="20"
+                height="20"
+                viewBox="0 0 24 24"
+                fill="none"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
```

Comment:
Can we save icons in the public folder and import them here using the Nextjs Image component for a cleaner look? 😉